presto-spi/src/main/java/io/prestosql/spi/StandardErrorCode.java

```diff
@@ -64,6 +64,47 @@
     INVALID_SPATIAL_PARTITIONING(0x0000_0029, USER_ERROR),
     INVALID_ANALYZE_PROPERTY(0x0000_002A, USER_ERROR),
     TYPE_NOT_FOUND(0x0000_002B, USER_ERROR),
+    CATALOG_NOT_FOUND(0x0000_002C, USER_ERROR),
+    SCHEMA_NOT_FOUND(0x0000_002D, USER_ERROR),
+    TABLE_NOT_FOUND(0x0000_002E, USER_ERROR),
+    COLUMN_NOT_FOUND(0x0000_002F, USER_ERROR),
+    ROLE_NOT_FOUND(0x0000_0030, USER_ERROR),
+    SCHEMA_ALREADY_EXISTS(0x0000_0031, USER_ERROR),
+    TABLE_ALREADY_EXISTS(0x0000_0032, USER_ERROR),
+    COLUMN_ALREADY_EXISTS(0x0000_0033, USER_ERROR),
+    ROLE_ALREADY_EXISTS(0x0000_0034, USER_ERROR),
+    NAMED_QUERY_ALREADY_EXISTS(0x0000_0035, USER_ERROR),
```

Comment:
This name is confusing since it's similar to the others, but means something different. How about `DUPLICATE_NAMED_QUERY`, similar to `DUPLICATE_PROPERTY`